### ISSUE TITLE
[NEW FEATURE] Add the possibility to skip CCDs in the analysis

### ISSUE THREAD

**Issue by aguinot** [CosmoStat/shapepipe#577]
We would like to investigate the impact of the lack of stars in the corner of the focal plane.

To do this we would need to discard some particular CCDs from the entire analysis. We first need to check if MCCD can handle an "incomplete" single exposure. Probably @tobias-liaudat have the answer to that. Then we can, for example, add a check in SETools that output an empty star catalogue for the flagged CCDs. Normally the rest of the pipeline should automatically deal with the missing PSF.

**Comment by tobias-liaudat** [comment]
@aguinot yes, MCCD can handle an "incomplete" exposure. It is actually the case right now, not all the exposures are complete, for example if a CCD has less than the minimum number of stars it will not be used and the exposure will be incomplete. If the input star catalogue received by MCCD does not include the problematic CCD, MCCD should work without problems.
